packages/@uppy/transloadit/src/index.js

```diff
@@ -188,14 +188,12 @@ export default class Transloadit extends BasePlugin {
     return newFile
   }
 
-  #createAssembly (fileIDs, uploadID, options) {
+  #createAssembly (fileIDs, uploadID, assemblyOptions) {
     this.uppy.log('[Transloadit] Create Assembly')
 
     return this.client.createAssembly({
-      params: options.params,
-      fields: options.fields,
```

Comment:
Isn't this a breaking change?

Reply:
No we still pass them, see the spreading below.